### ISSUE TITLE
Price notifications / alerts / blink

### ISSUE THREAD

**Issue by FelikZ** [unspentio/ethereum-gas-prices-browser-extension#2]
Dear Author,

Thank you for the great and yet minimalistic extension!

As feature request, I would like to add any preferred method to get notified when a certain value within a range:
- popup notification
- alert
- icon blinking
- any!

That would be handy to not manually check gas fees but automate it a little.
And with a privacy in mind (other extensions are not doing that)

Thank you anyway!

**Comment by philippe-git** [comment]
Yes that would be neat! If I ever have time to do it I will, but until this issue is closed, anyone stopping by here and having the skills is welcome to implement something along those lines after discussing in this thread about the best UX!
